src/Services/apiEndpoints.ts

```diff
@@ -0,0 +1,13 @@
+import { gql } from '@apollo/client';
```

Comment:
I suggest removing this file.

Reply:
Resolved on 583001d167d9742054bd0cf9dd64e843bb5b8a7d